www/modal.html

```diff
@@ -0,0 +1,119 @@
+
+        <meta name="theme-color" content="#ffffff">
+        <meta name="msapplication-TileColor" content="#ffffff">
+        <link rel="manifest" href="site.webmanifest">
+        <link rel="apple-touch-icon" sizes="180x180" href="assets/images/favicons/apple-touch-icon.png">
+        <link rel="icon" type="image/png" sizes="32x32" href="assets/images/favicons/favicon-32x32.png">
+        <link rel="icon" type="image/png" sizes="16x16" href="assets/images/favicons/favicon-16x16.png">
+        <link rel="mask-icon" href="assets/images/favicons/safari-pinned-tab.svg" color="#000000">
+
+        <link id="main-css" rel="stylesheet" href="assets/styles/main.css" media="print" onload="this.media='all'; this.onload=null; this.isLoaded=true">
+        <style type="text/css">
+            .o-link-like {
+                background-color: transparent;
+                text-decoration: underline;
+                border: 0;
+                margin: 0;
+                padding: 0;
+                font: inherit;
+                cursor: pointer;
+                color: #1A0DAB;
+            }
+
+            .o-link-like:hover,
+            .o-link-like:active {
+                color: #13097C;
+            }
+        </style>
+    </head>
+    <body data-module-load>
+        <div data-load-container>
+            <div class="o-scroll" data-module-scroll="main">
+                <header data-scroll-section>
+                    <a href="/"><h1>Locomotive Boilerplate</h1></a>
+                    <nav>
+                        <ul>
+                            <li><a href="images.html">Images</a></li>
+                            <li><a href="form.html" data-load="customTransition">Form</a></li>
+                            <li><a href="modal.html">Modal</a></li>
+                        </ul>
+                    </nav>
+                </header>
+
+                <main data-scroll-section>
+                    <div class="o-container">
+                        <h1 class="c-heading -h1">Modal</h1>
+
+                        <p>
+                            This page demonstrates our Modal component that enhances Kitty Giraudel's
+                            <a href="https://github.com/HugoGiraudel/a11y-dialog">a11ty-dialog</a>
+                            script to create accessible dialog windows.
+                        </p>
+                        <p>
+                            To see this dialog in action, you just need to
+                            <button class="o-link-like" data-a11y-dialog-show="my-dialog">open the dialog window</button>.
+                            Once it’s open, you should not be able to interact with other links
+                            on the main page. The focus is said to be “trapped” inside the dialog
+                            until the user explicitely decides to leave it.
+                        </p>
+                    </div>
+                </main>
+
+                <div class="c-dialog_container"
```

Comment:
I would advice to put the markup of `c-dialog_container` outside the `<div class="o-scroll">`. If no `data-scroll-section` are specified, then the o-scroll "becomes" the scroll-section and a transform will be applied on the element preventing all children with position `fixed` to work properly